config/config.yml

```diff
@@ -7,37 +7,38 @@ defaults: &defaults
   master_file_service_url: <%= ENV["PMP_MASTER_FILE_SERVICE_URL"] || "http://purl.dlib.indiana.edu/iudl/variations/master" %>
   access_file_service_url: <%= ENV["PMP_ACCESS_FILE_SERVICE_URL"] || "http://purl.dlib.indiana.edu/iudl/variations/access" %>
   purl_redirect_url: <%= ENV["PMP_PURL_REDIRECT_URL"] || "http://server1.variations2.indiana.edu/cgi-bin/access?%s" %>
+  create_ocr_files: <%= ENV["PMP_CREATE_OCR_FILES"] || true %>
 
   jp2_recipes:
     default_color: >
-      -rate 2.4,1.48331273,.91673033,.56657224,.35016049,.21641118,.13374944,.08266171 
```

Comment:
These "changes" all appear to be editor induced white space changes. I don't see any lines that have been altered. No changes were intended.